src/components/InjectedComponents/TypedMessageRenderer.tsx

```diff
@@ -77,13 +94,12 @@ export const DefaultTypedMessageComplexRenderer = React.memo(function DefaultTyp
         }
     }
     const R = props.TypedMessageRenderer || DefaultTypedMessageRenderer
-    return renderWithMetadata(
-        props,
+    return (
         <>
```

Comment:
This a React.Fragment can removed?

see https://reactjs.org/docs/fragments.html#keyed-fragments

Reply:
This component will be used as a child node of other components.